### ISSUE TITLE
Ocean placement next to another ocean not giving the $ bonus

### ISSUE THREAD

**Issue by GeoffreyHull** [terraforming-mars/terraforming-mars#6856]
I placed an ocean tile next to 2 other ocean tiles by using Large Convoy. I did not get the $4 bonus.
![IMG_0321](https://github.com/user-attachments/assets/8072885e-3e68-4f38-8cb0-658112cef827)
![IMG_0320](https://github.com/user-attachments/assets/314e08fb-64fc-4dee-ac8e-248a1082ecc1)

Remember that the preferences gear has a feature for reporting bugs which provides useful game data for us to try to reproduce issues.

{
  "url": "https://terraforming-mars.herokuapp.com/spectator?id=s3de2b410ffd8",
  "color": "orange",
  "step": 188,
  "version": "0ddc963",
  "builtAt": "Sat, 20 Jul 2024 01:36:34 GMT",
  "browser": "Safari 17",
  "language": "en",
  "experimental_ui": false
}

**Comment by GeoffreyHull** [comment]
I didn’t pay much attention to my total before and after so that’s likely what happened. To build onto what you said about logs, I would love for the log to have all changes to anything spelled out. As a new player it can be hard to follow the affects of each card/action. Especially in an asynchronous session where I log back in and have to track each card and action to learn exactly what the outcomes were. 

I can look into contributing back with something like different log levels? “Verbose” spelling out every change explicitly?

**Comment by kberg** [comment]
The two problems are 1) storage space and 2) information fatigue. But I'm looking.
